VSRAD.Package/Commands/EvaluateSelectedCommand.cs

```diff
@@ -19,10 +19,12 @@ sealed class EvaluateSelectedCommand : BaseRemoteCommand
         private readonly IProject _project;
         private readonly IActiveCodeEditor _codeEditor;
         private readonly QuickInfoEvaluateSelectedState _quickInfoState;
+#pragma warning disable IDE0052 // Remove unread private members
         private readonly IFileSynchronizationManager _deployManager;
         private readonly IOutputWindowManager _outputWindow;
         private readonly ICommunicationChannel _channel;
         private readonly IErrorListManager _errorListManager;
```

Comment:
Do you really need unread fields here?

Reply:
`EvaluateSelectedCommand` is one of the many obsolete parts of our codebase. It is not used anymore, the whole class should just be deleted, along with related components in `VSRAD.Package.ProjectSystem.EditorExtensions`.

Cleaning it up is in our backlog, but it's not a part of this PR, since as you correctly noted it's already way too big. The changes that were made are necessary to temporarily suppress a compile warning and remove dependencies on parts that were changed.